vspace/src/main/webapp/WEB-INF/views/layouts/navbar.html

```diff
@@ -1,46 +1,46 @@
-		  				<button style="background-color: #ffffff !important;" class="btn nav-link" type="submit" title="Logout" ><span style="color: #e39e44 !important;" class="icon-sign-out"></span></button>
-		         	</form>
-		         	<a class="btn nav-link" sec:authorize="isAnonymous()" th:href="@{/login}"><span class="icon-sign-in"></span></a>
-		        </div>
-		      </div>
-		    </nav>
-	    </div>
-	</body>
+<html layout:decorate="~{layouts/main}">
+	<head>
+	
+	</head>
+	
+	<body>
+		<div th:fragment="navbar">
+			<nav class="nav-bar" style="height:48px">
+		      <a class="navbar-brand" title="Go to exhibition" th:href="@{/}"><img th:src="@{/resources/maintenance/img/logo-11.png}" alt="Virtual Spaces logo" width="100px">
+		      [[${exhibition?.title}]]</a>
+		      <!-- <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+		        <span class="navbar-toggler-icon"></span>
+		      </button> -->
+		      <div class="nav" id="navbarCollapse">
+		        <ul class="navbar-nav mr-auto" style="list-style: none; display: contents;">
+		          <li class="nav-item active" style="padding-top: 8px;padding-right: 35px;">
+		            <a class="nav-link" th:href="@{/}" title="Go to exhibition"><span class="icon-play-circle secondary"></span>
+                        <span class="sr-only">(current)</span>
+                    </a>
+		          </li> 
+				  <li sec:authorize="hasAnyRole('ROLE_STAFF','ROLE_ADMIN')" class="nav-item active" style="padding-top: 8px;padding-right: 35px;">
+					<a class="nav-link" th:href="@{/staff/dashboard}" title="Staff Dashboard"><span class="icon-home secondary"></span>
+						<span class="sr-only">(current)</span>
+					</a>
+				  </li>
+                  
+                 <!--  <li class="nav-item active" style="padding: 15px;">
+                    <a class="nav-link" th:href="@{/}"><i class="far fa-play-circle secondary"></i></a>
+                  </li> 
+                  <li class="nav-item active" style="padding: 15px;">
+                    <a class="nav-link" th:href="@{/staff/exhibit/config}"><i class="fa fa-cog secondary"></i></a>
+                  </li>  -->
+                  
+		        </ul>
+		        <div class="form-inline mt-2 mt-md-0">
+		          	<form sec:authorize="isAuthenticated()" th:action="@{/logout}" method="POST">
+		         	 	<input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
+		  				<button style="background-color: #ffffff !important;" class="btn nav-link" type="submit" title="Logout" ><span style="color: #e39e44 !important;" class="icon-sign-out"></span></button>
+		         	</form>
+		         	<a class="btn nav-link" sec:authorize="isAnonymous()" th:href="@{/login}"><span class="icon-sign-in"></span></a>
+		        </div>
+		      </div>
+		    </nav>
+	    </div>
+	</body>

```

Comment:
It looks like you changed the line endings to windows line endings, please revert.